tests/__snapshots__/ResponsiveProvider.test.js.snap

```diff
@@ -357,29 +357,29 @@ Object {
   "greaterThan": Object {
     "_initial": true,
     "lg": false,
-    "md": true,
+    "md": false,
     "sm": true,
     "xl": false,
     "xs": true,
   },
   "is": Object {
     "_initial": false,
-    "lg": true,
-    "md": false,
+    "lg": false,
+    "md": true,
     "sm": false,
     "xl": false,
     "xs": false,
   },
   "isCalculated": true,
   "lessThan": Object {
     "_initial": false,
-    "lg": false,
+    "lg": true,
     "md": false,
     "sm": false,
     "xl": true,
     "xs": false,
   },
-  "mediaType": "lg",
+  "mediaType": "md",
```

Comment:
This file should reflect the same as before however I just noticed that before 1024 was `lg` which is weird, `md` should be the correct value here